Makefile

```diff
@@ -133,23 +133,16 @@ generate-stash-box-client:
 fmt:
 	go fmt ./...
 
-# Ensures that changed files have had gofmt run on them
-.PHONY: fmt-check
-fmt-check:
-	sh ./scripts/check-gofmt.sh
-
-# Runs go vet on the project's source code.
-.PHONY: vet
-vet:
-	go vet -mod=vendor ./...
-
 .PHONY: lint
-lint:
-	revive -config revive.toml -exclude ./vendor/...  ./...
+lint: golangci-lint
+
+.PHONY: golangci-lint
+golangci-lint:
+	golangci-lint run
```

Comment:
This is redundant. Just merge the `golangci-lint` target into `lint`.

Reply:
Yeah, I was thinking the same when I wrote it, but decided against it if more linters were incoming. But I guess we could just coalesce it for now. I'll push a change.